README.md

```diff
@@ -209,6 +209,40 @@ now any music downloaded to the **music** category will be imported by betanin
 
 <hr>
 
+### slskd
+
+create a script named `done.sh` or anything you like, and make it
+executable:  
+`chmod +x done.sh`
+
+modify the slskd configuration and add the following :
+
+```yml
+integration:
+  scripts:
+    notify_betanin:
+      on:
+        - DownloadDirectoryComplete
+      run: /path/to/done.sh $DATA
+```
+
+###### done script
+
+```bash
+#!/bin/bash
+
+read -t5 -r line
+path=$(echo "$line" | awk -F'"localDirectoryName":"' '{print $2}' | awk -F'"' '{print $1}' | awk '{printf "%s", $0}')
```

Comment:
i haven't tested, but would this work?

```suggestion
path="$(jq ".localDirectoryName")"
```

no need for `read` either since jq will receive stdin 

might need to add jq to the docker image too

Reply:
actually, it looks like the json comes as an argument to the script, not stdin

Reply:
I did a lot of try and error to make it work using only the tools included in the official docker image

Reply:
sure, but I think adding jq as a dependency here https://github.com/sentriz/betanin/blob/master/Dockerfile#L38 would be fine. i think it's worth it to parse the json propertly incase the directory path includes special chars like `"`

Reply:
I don't think adding jq to the betanin docker image would help ^^

(the script is run by slskd)

Reply:
I could make a pr on the slskd Dockerfile : https://github.com/slskd/slskd/blob/master/Dockerfile
But I don't know if it would be accepted.

Reply:
ahhaha oh yes, you're right, ofc :grin:

Reply:
https://github.com/slskd/slskd/pull/1294